src/main/java/seedu/duke/userinterface/CliUserInterface.java

```diff
@@ -1,4 +1,47 @@
 package seedu.duke.userinterface;
 
+import seedu.duke.exceptions.InvalidCommandException;
+import seedu.duke.tasks.TaskList;
+import seedu.duke.userinterface.command.Add;
+
+import java.lang.String;
+import java.util.Scanner;
+
 public class CliUserInterface {
+    private static int numberOfTasks;
+    private static TaskList list;
+    public Add add;
+
+    public CliUserInterface(TaskList list, int numberOfTasks) {
+        this.list = list;
+        this.numberOfTasks = numberOfTasks;
+    }
+
+    public void executeCommand(String userInput) throws Exception {
+        String[] input = userInput.trim().split(" ", 2); // split input into command and arguments
+        final String commandWord = input[0];
+        final String argument = userInput.replace(commandWord, "").trim();
```

Comment:
Could we just use `input[1]` for this?

Reply:
... Omg that is true HAHAHA okay, I'll change that.